ui/apps/everest/src/components/cluster-form/resources/resources.tsx

```diff
@@ -259,14 +264,25 @@ const ResourcesToggles = ({
             name={customNrOfUnitsInputName}
             textFieldProps={{
               type: 'number',
+              inputProps: {
+                step: dbType !== DbType.Mongo ? 1 : 2,
```

Comment:
@fabio-silva  this was expected, @recharte asked me yesterday to have only odd numbers for mongoDb custom nodes, so when i fixed this one, i checked that all default values in this field are odd, so by default user will open it with odd numbers and it will be increased by 2, which should only show odd numbers

Reply:
@solovevayaroslavna what I mean is that this will affect both proxy number and node number.
So, is that the expected behaviour, or just proxies?

Reply:
For MongoDB we want odd numbers for both DB nodes and proxies.

Reply:
got it. problem solved then